base/Dockerfile

```diff
@@ -22,11 +22,10 @@ RUN apk --no-cache add curl \
                        lm_sensors \
                        netcat-openbsd \
                        nodejs \
-                       py-mysqldb \
-                       py-psycopg2 \
                        py-yaml \
```

Comment:
For consistency let's install other Python packages via `pip install` too?

Reply:
Sure, sounds good to me